examples/InspectLibraries/main.go

```diff
@@ -0,0 +1,33 @@
+//
```

Comment:
Can you keep your package names all lower case? Use underscore as separator

Reply:
Can you give me a little more details?